templates/zerver/help/add-emoji.md

```diff
@@ -11,7 +11,6 @@ built-in emoji menu, pasting Unicode, or typing the emoji code.
 
```

Comment:
You can probably edit the previous content (lines 1-10) to fit to the standards a bit more.